src/tasks/buildTaskProvider.ts

```diff
@@ -0,0 +1,87 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { CommandTaskProvider, TitaniumTaskBase, TitaniumTaskDefinitionBase, TitaniumBuildBase } from './commandTaskProvider';
+import { selectBuildTarget } from '../quickpicks/common';
+import { TaskExecutionContext, Platform, ProjectType } from './tasksHelper';
```

Comment:
<!--
  0 failure: 
  2 warning:  src/tasks/buildTa..., src/tasks/buildTa...
  
  
  DangerID: danger-id-Danger;
  File: src/tasks/buildTaskProvider.ts;
  Line: 5;
-->

- :warning: src/tasks/buildTaskProvider.ts line 5 – 'Platform' is defined but never used. (@typescript-eslint/no-unused-vars)
- :warning: src/tasks/buildTaskProvider.ts line 5 – 'ProjectType' is defined but never used. (@typescript-eslint/no-unused-vars)